lib/middleware/verify-admin.js

```diff
@@ -0,0 +1,13 @@
+const User = require('../models/User');
+
+module.exports = (req, res, next) => {
+  const token = req.cookies.session;
+  User
+    .findByToken(token)
+    .then(user => {
+      if(user.role === 'admin'){
+        next();
+      }
```

Comment:
you need to call next with an error message or throw an error if the user isn't an admin